tools/test/audio/eqfir_run.sh

```diff
@@ -11,4 +11,4 @@ fi
 COMP=eq-fir
 DIRECTION=playback
 
-./comp_run.sh $COMP $DIRECTION $1 $2 $3 $3 $4 $5
+./comp_run.sh $COMP $DIRECTION "$1" "$2" "$3" "$3" "$4" "$5"
```

Comment:
See above, etc.

Reply:
Argument 3 (sample rate) is duplicated. I though it's not necessary to provide output and input rate separately as it's done with rate conversion components.